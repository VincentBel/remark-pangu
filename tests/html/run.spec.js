runSpec(__dirname);
runSpec(__dirname, {
  type: {
    html: false,
  },
});
