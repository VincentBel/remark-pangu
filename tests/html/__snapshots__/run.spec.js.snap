// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`html.md with options: {"link":false}: html.md 1`] = `
<div>abc123</div>

<div>我a我1</div>

<img src="https://a.com/中文a" alt="中文a" />
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<div>abc123</div>

<div>我a我1</div>

<img src="https://a.com/中文a" alt="中文a" />

`;

exports[`html.md with options: undefined: html.md 1`] = `
<div>abc123</div>

<div>我a我1</div>

<img src="https://a.com/中文a" alt="中文a" />
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<div>abc123</div>

<div>我a我1</div>

<img src="https://a.com/中文a" alt="中文a" />

`;
